client/src/app.jsx

```diff
@@ -46,23 +49,52 @@ class App extends React.Component {
           product_id: results[3].data.id,
           productInformation: results[3].data,
           styles: results[4].data,
-          qNa: this.props.qNaTestData
+          qNa: results[6].data,
+          savedQnA: results[6].data
         });
+
       })
       .catch((err) => {
         console.log('this is the err 🥲 ', err)
       });
   }
 
-  searchQuestionHandler(newState) {
-    console.log(newState)
-    this.setState({
-      qNa: newState
-    })
+  componentDidUpdate(prevProps, prevState) {
+    console.log(prevState.product_id)
+    console.log(this.state.product_id)
+    if (prevState.product_id !== this.state.product_id) {
+      Promise.all([
+        // REVIEW REQUESTS
+        reviews(), reviewsMeta(),
```

Comment:
I will remove the review calls in a future update, since they are self-contained in the reviews component.
For now this is good